backend/custom_models/item.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from .item_type import ItemType
+
+

```

Comment:
Can you just add simple comments in order to explain what's the goal/role of the model?
Please do the same for all the other models you added.

Reply:
Added in dd37aae